README.md

```diff
@@ -92,15 +92,15 @@ For project using Scala 2.11
 <dependency>
     <groupId>za.co.absa</groupId>
     <artifactId>atum_2.11</artifactId>
-    <version>3.5.1</version>
+    <version>ATUM_VERSION_HERE</version>
```

Comment:
👍 
Can we add a maven badge with the latest version to the README?

Reply:
That's a good idea, I will attempt to add it

Reply:
Done, looks nice: https://github.com/AbsaOSS/atum/blob/26ecc1fcfd275edd73df3fba6bab8043e7201000/README.md